plugins/network-elements/netscaler/src/com/cloud/network/element/NetscalerElement.java

```diff
@@ -512,7 +512,11 @@ public void doInTransactionWithoutResult(TransactionStatus status) {
         });
         HostVO host = _hostDao.findById(lbDeviceVo.getHostId());
 
-        _agentMgr.reconnect(host.getId());
+        try {
+            _agentMgr.reconnect(host.getId());
+        } catch (Exception e ) {
```

Comment:
Using a more specific catch makes sense, I will take this up in a separate PR. This was supposed to be a small improvement to the existing code.